composer.json

```
@@ -4,33 +4,34 @@
     "type": "sulu-bundle",
     "description": "Bundle that provides controllers and services for using Sulu as headless content management system",
     "require": {
-        "php": "^7.3 || ^8.0",
-        "sulu/sulu": "^2.4 || ^2.5@dev",
-        "symfony/config": "^4.4 || ^5.0 || ^6.0",
-        "symfony/dependency-injection": "^4.4 || ^5.0 || ^6.0",
-        "symfony/framework-bundle": "^4.4 || ^5.0 || ^6.0",
-        "symfony/http-foundation": "^4.4 || ^5.0 || ^6.0",
-        "symfony/http-kernel": "^4.4 || ^5.0 || ^6.0"
+        "php": "^8.2 || ^8.3",
+        "sulu/sulu": "^2.6@dev",
```

```suggestion
        "php": "^8.0",
        "sulu/sulu": "^2.5 || ^2.6@dev",
```

Sulu 2.5 is still a supported version with support for PHP 8.0 for the bundles where possible, which the SuluHeadlessBundle should be we not create 2 seperate versions for supporting Sulu 2.5 and 2.6 so the latest HeadlessBundle version should still support that range.